packages/react-devtools-core/src/standalone.js

```diff
@@ -256,7 +256,7 @@ function initialize(socket: WebSocket) {
 
   store = new Store(bridge, {
     checkBridgeProtocolCompatibility: true,
-    supportsNativeInspection: false,
+    supportsNativeInspection: true,
```

Comment:
One other place the `standalone` is used in Safari. Could you test there as well to make sure inspection works there as well?

Reply:
Just tested on a react app on Safari, inspection works!